retry.go

```diff
@@ -178,18 +177,21 @@ func (r *realisClient) thriftCallWithRetries(thriftCall auroraThriftCall) (*auro
 			case aurora.ResponseCode_OK:
 				return resp, nil
 
-				// If the response code is transient, continue retrying
+			// If the response code is transient, continue retrying
 			case aurora.ResponseCode_ERROR_TRANSIENT:
 				r.logger.Println("Aurora replied with Transient error code, retrying")
 				continue
 
-				// Failure scenarios, these indicate a bad payload or a bad config. Stop retrying.
+			// Failure scenarios, these indicate a bad payload or a bad config. Stop retrying.
 			case aurora.ResponseCode_INVALID_REQUEST:
+				fallthrough
```

Comment:
why not use case list instead of fallthrough?

Reply:
Can you give me an example of what this would look like?

Reply:
Just saw what this looks like, I'll update the PR

Reply:
you can use multiple condition in one case 

````go
switch nitendo {
case Mario, Zelda:
      // ....
}
````

see playground https://play.golang.org/p/XHnWbHeq814